pkg/resourcesets/collector.go

```diff
@@ -134,11 +134,20 @@ func (h *ResourceHandler) gatherResourcesForGroupVersion(filter v1.ResourceSelec
 	// "resources" list has all resources under given groupVersion, first filter based on KindsRegexp
 	if filter.KindsRegexp != "" {
 		if filter.KindsRegexp == "." {
-			// "." will match anything, so return entire resource list
-			return resources.APIResources, nil
+			// "." will match anything expect excludeKinds, so return entire resource list
+			for _, res := range resources.APIResources {
+				if slice.ContainsString(filter.ExcludeKinds, res.Kind) {
```

Comment:
Although the value of Kind is in the PascalCasing format (capitalizing the first letter of each word) by design, should we consider some variations in users' input? For example, "Token" vs "token", "ConfigMap" vs "configMap" vs "configmap". 

A possible solution for it is to capitalize values of `filter.ExcludeKinds` and `res.Kind` before comparing. 

FYI, we support both PascalCasing and all lower letters in the `kindsRegexp`.

Reply:
I think we're case-sensitive in all other cases, so it would be weird that we're case insensitive here. I do see that we match `KindsRegexp` to both `res.Kind` and `res.Name`, so should we check both of these against `filter.ExcludeKinds` if the latter is specified?

Another point towards this is how weird this looks in config:
```yaml
...
excludeKinds:
    - "Token"
- apiVersion: "management.cattle.io/v3"
  kindsRegexp: "^tokens$"
...
```
So we have `excludeKinds` set as `Token` and right below it we have `kindsRegexp` as `^tokens$` making it look quite inconsistent.

Reply:
I can make it case-insentive. But I agree with @snasovich that it does looks weird on the config, but there is no good way to make an exception match for a type today

Reply:
@StrongMonkey , I was suggesting to have `excludeKinds` act on `res.Name` as well to match behavior of `kindsRegexp`. Still a bit questionable behavior, but it would make it more consistent and allow us to configure.

On a side note we could do `excludeKindsRegexp` string to have more flexibility compared to current `excludeKinds` array. But I'll leave it to you to decide if we want to change to it.

Reply:
@snasovich @jiaqiluo ok, I added support for both kind and plural names so it looks more consistent now. And the config is also changed to use plural name of token. I will skip the implementation of `excludeKindsRegexp` for now since it is just a hot fix and I would like to avoid any feature work that increase more complexity, but I do agree that is a useful feature if we are going to add it later.